src/pwncore/models/ctf.py

```diff
@@ -30,14 +32,29 @@ class Problem(BaseProblem):
         null=True
     )  # type: ignore[assignment]
 
+    mi = fields.IntField(default=100)  # Arbitrary meaning full defaults
+    ma = fields.IntField(default=600)
+
     hints: fields.ReverseRelation[Hint]
 
+    class PydanticMeta:
+        exclude = ["image_name", "image_config"]
+
+    async def _solves(self) -> int:
+        return await SolvedProblem.filter(problem=self).count()
+
+    async def update_points(self) -> None:
+        self.points = round(
+            self.mi + (self.ma - self.mi) * (1 - tanh(await self._solves()))
```

Comment:
Need to add the constant m (Maybe a sync with the master branch shall pull the changes merged from #13)

Reply:
I will handle while resolving merge conflicts